store/src/java/com/zimbra/cs/index/LuceneIndex.java

```diff
@@ -960,9 +986,15 @@ public void close() throws IOException {
             if (count.decrementAndGet() == 0) {
                 ZimbraLog.search.debug("Close IndexSearcher");
                 try {
-                    Closeables.closeQuietly(luceneSearcher);
+                    try {
```

Comment:
Would feel happier here with a replacement like closeQuietly in case vars are null?